### ISSUE TITLE
CodeMirror is not defined - Angular 9

### ISSUE THREAD

**Issue by schuchard** [TypeCtrl/ngx-codemirror#263]
I was unable to get this library running given the current steps for Angular 9 and  I wanted to share this in case anyone else ran across this error. 

It appears that the global CodeMirror is not available when the component is bootstrapped, given the current setup steps. Specifically, I think `codemirror/lib/codemirror.js` needs to be loaded as well and that's what eventually got everything working. I ended up adding the codemirror files to my angular.json and not the main.ts and *.css file.

```json
"styles": [
  "./node_modules/codemirror/lib/codemirror.css",
  "./node_modules/codemirror/theme/material.css"
],
"scripts": [
  "./node_modules/codemirror/lib/codemirror.js",
  "./node_modules/codemirror/mode/javascript/javascript.js"
],
```

I'm using the following versions:

```json
"@ctrl/ngx-codemirror": "3.1.0",
"codemirror": "5.53.2",
"@angular/core": "~9.1.0",
```

Feel free to close this issue if you're unable to repro with Angular 9. Thanks for maintaining this library.

**Comment by scttcper** [comment]
reverted the recent change, thanks for opening this issue https://github.com/TypeCtrl/ngx-codemirror/commit/7297faeaa6d78b2ca3b14d7f1c29b7dc85c25592

**Comment by scttcper** [comment]
Should be fixed in 3.1.1 https://github.com/TypeCtrl/ngx-codemirror/releases/tag/v3.1.1

**Comment by mikenlanggio** [comment]
I'm using Angular 12 in facing exactly the same with this issue.
@scttcper any help?
Thank you
